_shows/20_21/accelerate.md

```diff
@@ -0,0 +1,64 @@
+---
+# Check out the Show Docs for more information
+# -- https://history.newtheatre.org.uk/docs/show/
+# Remove or comment any lines that are not being used
+
+title: Accelerate
+playwright: Bill Hayes
+student_written: true
+period: Autumn
+season: Online
+season_sort: 440
+date_start: 2020-12-16
+date_end: 2020-12-16
+
+cast:
+- role: Mason
+  name: Jake Levy
+- role: Sadie
+  name: Daisy Miles
+- role: Piig
```

Comment:
Is this a typo?

Reply:
> 



> Lots of little comments here. Pretty solid though! Throughout, when referencing show titles in the "watch show on YouTube", can the show title be in single quotes please?

Yes will do! And yes, the character's name was Piig